src/Logging.jl

```diff
@@ -126,7 +126,7 @@ write_log(output::Base.TTY, color::Symbol, msg::AbstractString) = (Base.println_
 function formatlog(format::AbstractString, logging_fields::Dict)
     function substr(s)
         v = get(logging_fields, s[3:end-1], "<none>")
-        return isa(v, Function) ? string(v()) : string(v)
+        return isa(v, Function) ? string(eval(Main,v())) : string(eval(Main,v))
```

Comment:
The reason I put it on both is I thought it might be useful to store the symbol (ie. `:REQUEST_ID`) and have it evaluate, but maybe that has some unintended consequences. I didn't really think it would hurt.

Let my play a bit more with the functions, I agree it is strange.. but for some reason it wasn't working until I added this.